app/controllers/wishlists_controller.rb

```diff
@@ -0,0 +1,22 @@
+class WishlistsController < ApplicationController
+  def new
+    @wishlist = Wishlist.new
+  end
+
+  def create
+    @wishlist = Wishlist.new(wishlist_params)
+    @wishlist.user_id = current_user.id
+    if @wishlist.save
+      flash[:notice] = "Success"
+      redirect_to dashboard_path
+    else
+      redirect_to new_wishlist_path, notice: @wishlist.errors.messages
```

Comment:
Similar note to above, consider using `render :new` instead of redirecting. Then you can display validation errors inline etc.